Obsidian/WorldData/Generators/SuperflatGenerator.cs

```diff
@@ -13,12 +13,11 @@ public override Chunk GenerateChunk(int x, int z, World world, Chunk chunk = nul
         {
             for (var z1 = 0; z1 < 16; z1++)
             {
-                chunk.SetBlock(x1, 5, z1, Registry.GetBlock(Material.GrassBlock));
-                chunk.SetBlock(x1, 4, z1, Registry.GetBlock(Material.Dirt));
-                chunk.SetBlock(x1, 3, z1, Registry.GetBlock(Material.Dirt));
-                chunk.SetBlock(x1, 2, z1, Registry.GetBlock(Material.Dirt));
-                chunk.SetBlock(x1, 1, z1, Registry.GetBlock(Material.Dirt));
-                chunk.SetBlock(x1, 0, z1, Registry.GetBlock(Material.Bedrock));
+                chunk.SetBlock(x1, -60, z1, Registry.GetBlock(Material.GrassBlock));
+                chunk.SetBlock(x1, -61, z1, Registry.GetBlock(Material.Dirt));
+                chunk.SetBlock(x1, -62, z1, Registry.GetBlock(Material.Dirt));
+                chunk.SetBlock(x1, -63, z1, Registry.GetBlock(Material.Dirt));
+                chunk.SetBlock(x1, -64, z1, Registry.GetBlock(Material.Bedrock));
```

Comment:
To be fair the superflat world gen could still use some optimization iirc

e.g. generate one chunk and copy it instead of building a new one each time

not necessarily commentary on this specific PR, could be made a new issue